exporter/jaeger/jaeger_test.go

```diff
@@ -49,3 +54,75 @@ func Test_bytesToInt64(t *testing.T) {
 		})
 	}
 }
+
+func Test_spanDataToThrift(t *testing.T) {
+	now := time.Now()
+
+	answerValue := int64(42)
+	resultValue := true
+	keyValue := "value"
+	statusCodeValue := int64(2)
+	statusMessage := "error"
+
+	tests := []struct {
+		name string
+		data *trace.SpanData
+		want *gen.Span
+	}{
+		{
+			name: "no parent",
+			data: &trace.SpanData{
+				SpanContext: trace.SpanContext{
+					TraceID: trace.TraceID{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 14, 15, 16},
```

Comment:
Could you make this a 16 byte TraceID to conform with the usual? This is 15 bytes and it even produces Spans that have 15 byte SpanIDs and TraceIDs.

Reply:
Aha, you missed 12 in there :)

Reply:
Whoops, it was a mistake. Done.

Reply:
That'll change the value of the TraceIDHigh, *Low, SpanID.
Let's also add tests to ensure that they TraceIDHigh is 8 bytes and same for TraceIDLow otherwise we'd never catch such discrepancies.

Reply:
I don't think we need a test for this. There might be trailing zeros. The trace ID was 16-bytes with a trailing zero which is still valid.